Controls/Grid/Grid.UWP/Model/TableModel/DisplayData/CellsController.cs

```diff
@@ -76,18 +76,20 @@ internal RadSize OnCellsArrange(RadSize finalSize)
             foreach (var rowPairs in this.RowPool.GetDisplayedElements())
             {
                 int rowSlot = rowPairs.Key;
+                var rowModel = rowPairs.Value.Last();
                 if (firstRow)
                 {
                     firstRow = false;
-                    int line = rowPairs.Value.Last().ItemInfo.Slot;
-                    topOffset = line - 1 >= 0 ? this.RowPool.RenderInfo.OffsetFromIndex(line - 1) : 0;
+                    int line = rowModel.ItemInfo.Slot;
+                    topOffset = line - 1 >= 0 ? (this.RowPool.RenderInfo.OffsetFromIndex(line - 1) - rowModel.RowDetailsSize.Height) : 0;
                 }
 
                 Dictionary<int, T> columnCellsPair;
                 this.generatedRowCells.TryGetValue(rowSlot, out columnCellsPair);
 
                 leftOffset = 0;
-                double cellHeight = this.GetSlotHeight(rowSlot);
+                double rowSlotHeight = this.GetSlotHeight(rowSlot);
```

Comment:
Do we need both rowSlotHeight and cellHeight - it seems the cellHeight is equal to the rowSlotHeight? Shouldn't we use only one of them as the previous implementation?